pyproject.toml

```
@@ -32,8 +32,8 @@ dependencies = [
   "backoff>=2.1.2,<2.3",  # Looks like only bugfixes in z-Stream.
   "click>=8.1.0,<8.3",  # Uses milestone.feature.fix https://palletsprojects.com/versions .
   "cryptography>=44.0.3,<46.0",  # SemVer compatible https://cryptography.io/en/latest/api-stability/#versioning .
-  "Django~=4.2.0",  # LTS version, switch only if we have a compelling reason to".
-  "django-filter>=23.1,<=25.1",  # Looks like CalVer.
+  "Django>=4.2.0,<=5.2.0",  # LTS version, switch only if we have a compelling reason to".
```

The current release is 5.2.6. But I think with Django we can stick to x.y giving us essentially:


`http https://pypi.org/pypi/Django/json | jq '.releases|keys'`
```suggestion
  "Django>=4.2.0,<5.3",  # LTS version, switch only if we have a compelling reason to".
```